webapi/src/main/resources/knora-ontologies/knora-base.ttl

```diff
@@ -2522,3 +2527,58 @@
                            "Sequence Bounds"@en ;
     rdfs:comment           "Deprecated in favour of :Segment"@en ;
     rdfs:comment           "Indicates the bounds of a sequence, i.e. the start and end point in the containing resource."@en .
+
+
+### http://www.knora.org/ontology/knora-base#TextValueType
+
+:TextValueType a owl:Class ;
+    owl:oneOf ( :UnformattedText
```

Comment:
You used it almost correctly (my bad), I will send you information about how to do it correctly.